python/cuml/test/test_dbscan.py

```diff
@@ -86,7 +86,7 @@ def test_dbscan_precomputed(datatype, nrows, max_mbytes_per_batch, out_dtype):
 
     # Precompute distances
     Xc = np.array([[complex(p[0], p[1]) for p in X]])
-    X_dist = np.abs(Xc - Xc.T, dtype=datatype)
+    X_dist = np.abs(Xc - Xc.T).astype(datatype)
```

Comment:
I know it's not part of this PR but the conversion to complex here seems to be creating issues. We could just compute the pairwise euclidean distance matrix here instead, perhaps.